### PR TITLE
remove references to enterprise

### PR DESCRIPTION
**Pull request by jademcgough**
Connect #490 

This removes most references to Enterprise, changing them to Chronograf.

We still have a `_enterprise-custom.scss` file, and the `/enterprise_style` directory. There's already a `/chronograf` directory for styles from the old chronograf project, so I wasn't sure how to rename it without being confusing.

### CODE REVIEW

File: ui/src/alerts/containers/AlertsApp.js [hunk 1]
```diff
@@ -103,9 +103,9 @@ const AlertsApp = React.createClass({
       // I stole this from the Hosts page.
       // Perhaps we should create an abstraction?
       <div className="hosts hosts-page">
-        <div className="enterprise-header">
-          <div className="enterprise-header__container">
-            <div className="enterprise-header__left">
+        <div className="chronograf-header">
+          <div className="chronograf-header__container">
+            <div className="chronograf-header__left">
```
Comment (alexpaxton) [inline ui/src/alerts/containers/AlertsApp.js]: @willpiers yeah I was thinking of doing something like this, saves me some time

**Comment by alexpaxton** [comment]
I'm wondering if it makes sense to rename the `Chonograf` directory to `Data Explorer` then we can have `/chonograf_style` and `_chronograf-custom.scss`

I think there is some renaming to do in the styles themselves, enterprise is referenced a few places

**Comment by alexpaxton** [comment]
@jademcgough actually it looks like you got everything CSS wise, baller

**Comment by jademcgough** [comment]
Is there anything missing from this, or can I merge?

**Pull request merged**